src/handlers/feature_handlers.cpp

```diff
@@ -1997,105 +2046,8 @@ void BridgeFerryHandler::Cull(int lod) {
 }
 
 void BridgeFerryHandler::Draw(int lod) {
-    auto bangerInst = reinterpret_cast<dgUnhitYBangerInstance*>(this);
```

Comment:
Why was this removed? Won't it stop bridges from being drawn?

Reply:
Ah I see. it's the "call original" which confused me, because the original Draw function does nothing.